[
  {
    "cmd": [
      "vpython3",
      "-u",
      "RECIPE_MODULE[recipe_engine::file]/resources/fileutil.py",
      "--json-output",
      "/path/to/tmp/json",
      "copy",
      "{\"key0\": true, \"key1\": true, \"key2\": true, \"key3\": true, \"key4\": true, \"key5\": true, \"key6\": true, \"key7\": true, \"key8\": true, \"key9\": true}",
      "[START_DIR]/some_file.json"
    ],
    "infra_step": true,
    "name": "write_json",
    "~followup_annotations": [
      "@@@STEP_LOG_LINE@some_file.json@{\"key0\": true, \"key1\": true, \"key2\": true, \"key3\": true, \"key4\": true, \"key5\": true, \"key6\": true, \"key7\": true, \"key8\": true, \"key9\": true}@@@",
      "@@@STEP_LOG_END@some_file.json@@@"
    ]
  },
  {
    "cmd": [
      "vpython3",
      "-u",
      "RECIPE_MODULE[recipe_engine::file]/resources/fileutil.py",
      "--json-output",
      "/path/to/tmp/json",
      "copy",
      "[START_DIR]/some_file.json",
      "/path/to/tmp/"
    ],
    "infra_step": true,
    "name": "read_json",
    "~followup_annotations": [
      "@@@STEP_TEXT@file command encountered system error JSON READ FAILURE@@@",
      "@@@STEP_FAILURE@@@"
    ]
  },
  {
    "failure": {
      "failure": {},
      "humanReason": "Step('read_json') failed 'JSON READ FAILURE' with: file command encountered system error JSON READ FAILURE"
    },
    "name": "$result"
  }
]